### ISSUE TITLE
A "wider" model?

### ISSUE THREAD

**Issue by RicardoPQ-com** [floe/backscrub#165]
Hi there. Does anyone have a model similar to "selfiesegmentation_mlkit-256x256-2021_01_19-v1215.f16.tflite", but that doesn't crop the sides of the image? I've tried "segm_lite_v509.tflite" and "segm_lite_v681.tflite", but they don't seem to have the same resolution as the first one.

**Comment by floe** [comment]
Hmmm, if I remember correctly, @BenBE had some code that would just run the model twice for wider images, but I'm not sure if that ever made it to the main branch...

**Comment by BenBE** [comment]
Nope. That code was some early WIP and I couldn't really get the necessary refactoring to work for processing multiple images per frame.
